src/goto-symex/symex_goto.cpp

```diff
@@ -216,7 +216,7 @@ void goto_symext::symex_goto(statet &state)
     return;
   }
 
-  // put into state-queue
+  // put a copy of the current state into the state-queue
   framet::goto_state_listt &goto_state_list =
     state.top().goto_state_map[new_state_pc];
 
```

Comment:
Maybe a slightly weird ask, but would you mind removing the blank line (line 222) here? The comment really only makes sense when also looking at the `goto_state_list.emplace_back(state);` instruction, but that's currently "the next paragraph."

Reply:
You've reminded me that I also wanted to give `state_pc` and `new_state_pc` better names. Hmmm.